### ISSUE TITLE
DPL-947 - As a long read user (Craig) I need the DNA control complex box barcode to be required on a v12 Sequel IIe run and included on the sample sheet because it is required for SMRT LInk and we are currently having to enter it manually

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-service#1129]
**User story**
As a long read user (Craig) I need the DNA control complex box barcode to be required on a v12 Sequel IIe run and included on the sample sheet because it is required for SMRT LInk and we are currently having to enter it manually

**Who are the primary contacts for this story**
Craig / Steve

**Who is the nominated tester for UAT**
Craig / Long Read

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] When a new v12 Sequel IIe run is created a DNA control complex box barcode can be added
- [ ] When a new v12 Sequel IIe run is created a DNA control complex box barcode should be required
- [ ] When a new v12 Sequel IIe sample sheet is created it should have a DNA control complex barcode
- [ ] SMRT Link Version v11 should be made inactive in the rake task and database for UAT and production
- [ ] Sample sheets should be sorted by plate and then wells [linked to another issue but so trivial not worth doing as a new story]

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
Closed as done.
